freespace2/freespace.cpp

```diff
@@ -7329,26 +6942,19 @@ DCF_BOOL( time_models, Time_models )
 
 void Do_model_timings_test()
 {
-	
-
 	if ( !Time_models ) return;
 
 	mprintf(( "Timing models!\n" ));
 
-	int i;
+	ubyte model_used[MAX_POLYGON_MODELS] = {0};
+	int model_id[MAX_POLYGON_MODELS] = {0};
```

Comment:
`model_id` doesn't need to be initialized here, but if it is, it should be -1, not 0